model/src/main/resources/reference.yml

```diff
@@ -150,6 +150,15 @@ ort:
 
         timeout: 60
 
+      ScanOss:
```

Comment:
Commit-message: 

There is a generic mechanism for ignoring files [1].
This raises the question to me whether that mechanism should be used, for the same reasoning it is being used for `ScanCode` currently. What do you think?

Apart from that, the below list of default values raises also the question whether the actual values are actually desired.
E.g. why should we exclude a build script from scanning, only for Scan OSS?

[1] https://github.com/oss-review-toolkit/ort/blob/2d27a2244f660e43b192058221805721033aa7ce/scanner/src/main/kotlin/Scanner.kt#L182-L184